### DISCUSSION TITLE
[re-enable] Feature Aircraft Picture

### DISCUSSION THREAD

**Discussion by ger-pakon** [vradarserver/vrs#53] [Ideas]
Hey,

I know that there were some issues with airport-data and some AV Scanners. Is it may possible to reactivate the feature?

In case of no, can you may supply some snipplet for the custom content plugin? I can't and won't revert to an older version.

Thank you and cheers, 
Pakon

**Comment by vradarserver** [comment]
I'm not sure who is running the server any more, emails to the guy who used to run it are no longer returned. So for the time being, no, it won't be coming back.

However, for those brave souls who want the thumbnails back there is a plugin in the **vrs-other-plugins** repository that reinstates them. You can find the latest builds of the plugin here:

Windows: https://github.com/vradarserver/vrs-other-plugins/releases/tag/latest-airportdata-thumbnails-windows
Mono: https://github.com/vradarserver/vrs-other-plugins/releases/tag/latest-airportdata-thumbnails-mono

**Reply by ger-pakon** [reply]
Thank you, according to some posts in their forum (https://www.airport-data.com/forums/topic10348.html and https://www.airport-data.com/forums/topic10347.html) its blacklistet. Indeed the running software is mostly outdated and could provide an atacker more room. :-/ (https://sitecheck.sucuri.net/results/airport-data.com)

Cheers,
Pakon

**Comment by Dretze** [comment]
Hey,
I'm using vrs v3.0.0-preview-8 [Linux / Mac OSX Mono] with the AirportDataThumbnails-mono-v3 plugin on a Raspberry Pi, but the aircraft picture are not displayed. On a Windows PC with vrs v3.0.0-preview-8 and the plugin-AirportDataThumbnails-v3 the picture are displayed.

The Raspberry Pi is running with:
Raspbian GNU/Linux 10 (buster)
Kernel 5.10.103-v7l+
Mono JIT compiler version 6.12.0.182 (tarball Tue Jun 14 22:53:32 UTC 2022)

Is there anything I need to consider when using vrs on a Raspberry Pi?

**Comment by ua4wiy** [comment]
If you have AV problems with airport-data, it may be possible to implement work with planespotters.net photo api?
https://www.planespotters.net/photo/api

**Reply by vradarserver** [reply]
That looks like it would work. I'll look into it a bit further, thanks.
